Test/overall_test.py

```diff
@@ -1492,4 +1492,38 @@
 2.6703554722378726
 >>> cm.sensitivity_index()[3]
 'None'
+>>> matrix = [[1,2,3],[4,6,1],[1,2,3]]
+>>> cm = ConfusionMatrix(matrix=matrix)
+>>> cm.print_matrix()
+Predict 0       1       2       
+Actual
+0       1       2       3       
+<BLANKLINE>
+1       4       6       1       
+<BLANKLINE>
+2       1       2       3       
+<BLANKLINE>
+<BLANKLINE>
+>>> cm = ConfusionMatrix(matrix=matrix, classes=["L1","L2","L3"])
+>>> cm.print_matrix()
+Predict  L1       L2       L3       
+Actual
+L1       1        2        3        
+<BLANKLINE>
+L2       4        6        1        
+<BLANKLINE>
+L3       1        2        3        
+<BLANKLINE>
+<BLANKLINE>
+>>> cm = ConfusionMatrix(matrix=matrix, classes=["L1","L3","L2"])
```

Comment:
@sadrasabouri 
As you can see in this example, classes order is not correct in the output.
I think you should make minor edits in the ` matrix_params_from_table` function to accept the ordered classes list!

Reply:
But I think that's OK. because we do the sorting for a reason (which I can not remember now :laughing:)

Reply:
So it would be for future PRs, right?
No changes needed for current PR?

Reply:
I think they are in the same scope and we can handle it in this PR.
But if you think it would lead to some bugs, tell us.

Reply:
The most successful idea would be to feature a new parameter (like `classes`) for `matrix_params_from_table` which provides us with an ordered list of classes (since it's impossible to keep order in dictionary keys - unless we redefine the table as a 2D list and a list for class_names). This parameter would be optional since there is some use in other parts of the code, so I think it's not best practice.

Reply:
Dictionary is pretty heavily optimized in Python. Creating a `dict` from N keys or key/value pairs is **O(N)**, fetching is **O(1)**, putting is amortized **O(1)**. So it seems is not an efficient method to redefine the table as a 2D list and a list for class_names!!

We have 2 scenarios:

1. Classes order is defined
2. Classes order is undefined

In the first scenario, the only way we can get the order from the user is a separate parameter (we called `classes`), so I think the best way is to inject `classes` into the `matrix_params_from_table` function as an optional parameter.